src/main/java/duke/storage/Storage.java

```diff
@@ -0,0 +1,100 @@
+package duke.storage;
+
+import java.io.IOException;
+import java.nio.charset.Charset;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+
+import duke.task.Task;
+import duke.task.ToDo;
+import duke.task.Deadline;
+import duke.task.Event;
+
+/**
+ * Class for reading and writing Duke data.
+ */
+public class Storage {
+    /**
+     * Charset for reading and writing .txt files.
+     */
+    final static Charset ENCODING = StandardCharsets.UTF_8;
+    /**
+     * Path for data.
+     */
+    private Path path;
```

Comment:
Do we need to Javadoc private attributes? They are not supposed to be visible to the outside world but should stay local in the class. Javadoc will reveal the actual implementation of this class, which breaks abstraction somewhat.